lib/routes/dev/dev.dart

```diff
@@ -606,12 +616,12 @@ class DevViewState extends State<DevView> {
       ),
     );
     choices.add(Choice(
-      title: marketplaceBloc.isVendorToggle == true
-          ? 'Display Snort'
-          : 'Hide Snort',
+      title: "${marketplaceBloc.isSnortToggle ? "Display" : "Hide"} Snort",
       icon: Icons.phone_android,
       function: () {
-        _toggleVendor(marketplaceBloc);
+        // NostrSettings nostrSettings = NostrSettings();
+        // the instance of nostrSettings is null correct it
```

Comment:
Isn't this addressed? It looks like you do set the default value as `NostrSettings.start()` when initializing `MarketplaceBloc`.